public-images/public-images-explanation/lxd-openstack-images.rst

```diff
@@ -0,0 +1,210 @@
+Images <https://cloud-images.ubuntu.com/minimal/>`__:
+Ubuntu images that have a reduced runtime footprint, optimised kernel
+and boot process. They are smaller and boot faster, but are not designed
+for environments requiring human interaction or debugging.
+
+
+LXD images
+~~~~~~~~~~
+
+To import an image into LXD, you will need two artefacts:
+
+1. An LXD tarball:
+
+   - The :ref:`lxd-tarball-ref` artefact has the extension ``*.lxd.tar.xz``
+   - It contains the metadata needed by LXD to instantiate a container or virtual machine as well as a folder for any custom templates
+
+2. A file system for a container or a bootable disk image for a virtual
+   machine:
+
+   - The file system for a container can be either a 
+
+     * :ref:`root-tarball-ref` (``*-root.tar.xz``), or a 
+
+     * :ref:`squashfs-ref` (``*.squashfs``)
+
+   - The bootable disk image for a virtual machine is a :ref:`qcow-ref` image (``*.img``)
+
+The following are example commands to import LXD containers and virtual machines, respectively, based on downloaded Ubuntu 24.04 artefacts:
+
+.. code :: bash
+
+   lxc image import noble-server-cloudimg-amd64-lxd.tar.xz \
+      noble-server-cloudimg-amd64-root.tar.xz --alias noble_container
+
+.. code :: bash
+
+   lxc image import noble-server-cloudimg-amd64-lxd.tar.xz \
+      noble-server-cloudimg-amd64.img --alias noble_vm
+
+
+OpenStack images
+~~~~~~~~~~~~~~~~
+
+OpenStack uses QCOW images. Download the artefact for your chosen
+architecture with the ``*.img`` extension.
+
+Use the `OpenStack command-line
+client <https://docs.openstack.org/ocata/admin-guide/common/cli-install-openstack-command-line-clients.html>`__
+to interact with OpenStack. An example of uploading an image looks 
+something like this:
+
+.. code :: bash
+
+   openstack image create “Ubuntu-24.04” \
+      --file noble-server-cloudimg-amd64.img \
+      --disk-format qcow2 \
+      --container-format bare \
+      --public
+
+To learn more about managing images with OpenStack, you can read their
+documentation
+`here <https://docs.openstack.org/ocata/admin-guide/common/cli-manage-images.html#create-or-update-an-image-glance>`__.
+
+
+Configuration
+-------------
+
+Configuring an Ubuntu cloud image allows you to make changes that tailor
+the image to your specific use case. You can automate the creation of
+user accounts, configure SSH access, or install software before the
+instance starts.
+
+
+LXD images
+~~~~~~~~~~
+
+You can configure your cloud images in LXD either before you import them
+or after. Configuring your images before importing them is most commonly
+done by editing the ``metadata.yaml`` file contained in the LXD tarball.
+Configuring your images after importing them is done through the CLI.
+
+If you are interested in configuration of LXD containers rather than
+images, take a look at the Ubuntu Server documentation
+`here <https://documentation.ubuntu.com/server/how-to/containers/lxd-containers/>`__.
```

Comment:
Could you do the same thing with the link here? (Change 'here' and move the link to the end of the file)